examples/seismic/abc_methods/02_damping.ipynb

```diff
@@ -0,0 +1,1236 @@
+{
```

Comment:
<p>again we should avoid importing everything from devito</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Done!